keyboards/handwired/oled25/readme.md

```diff
@@ -0,0 +1,16 @@
+# oled25
```

Comment:
Could you align this, and add content based on <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md#bootloader>

Reply:
I transferred my description into the template so it should be correct now. 
I'm not sure if the way I have the supported hardware and hardware availability filled out correctly for a handwired keyboard so please let me know if it needs to be changed.